data/templates/centos-ks

```diff
@@ -45,14 +45,14 @@ clearpart --all --drives=<%=installDisk%>
         <% if(partition.mountPoint !== undefined && partition.size !== undefined) { %>
             <% if(partition.fsType !== undefined) { %>
                 <% if(partition.size === 'auto') { %>
-                    partition <%=partition.mountPoint%> --grow --fstype=<%=partition.fsType%>
+                    partition <%=partition.mountPoint%> --size=250 --grow --fstype=<%=partition.fsType%>
```

Comment:
--size is required for centOS 6, the install will fail if --size is not specified.  Since --grow is also specified it will configure the partition to use all available space with 250 being the min.